Plugins/Gerrit/FormGerritPublish.cs

```diff
@@ -26,6 +27,7 @@ public FormGerritPublish(IGitUICommands uiCommand)
         {
             InitializeComponent();
             Translate();
+            Trace.Assert (toolTip1 != null); // suppress warning on mono, because of "error as warning"
```

Comment:
I would rather suggest to remove this variable completly since I can't see where it would be used.